server/services/address.js

```diff
@@ -72,7 +72,8 @@ module.exports = {
       console.log('About to: getPostcodeFromEastingorNorthing')
       const uri = `${config.placeApi?.url}?point=${easting},${northing}&key=${config.ordnanceSurvey.osSearchKey}`
       const payload = await util.getJson(uri)
-      if (!payload || !payload?.results[0]) {
+      console.log(payload.results)
```

Comment:
did you mean to leave this log in?

Reply:
no - good spot. This request was implemented by the previous dev, it was not returning data and causing a crash.